dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/MetadataFieldRestRepository.java

```diff
@@ -98,4 +113,112 @@ public MetadataFieldRest findOne(Context context, Integer id) {
     public MetadataFieldResource wrapResource(MetadataFieldRest bs, String... rels) {
         return new MetadataFieldResource(bs, utils, rels);
     }
+
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
+    protected MetadataFieldRest createAndReturn(Context context)
+            throws AuthorizeException, SQLException {
+
+        // parse request body
+        MetadataFieldRest metadataFieldRest;
+        try {
+            metadataFieldRest = new ObjectMapper().readValue(
+                    getRequestService().getCurrentRequest().getHttpServletRequest().getInputStream(),
+                    MetadataFieldRest.class
+            );
+        } catch (IOException excIO) {
+            throw new PatchBadRequestException("error parsing the body ..." + excIO.getMessage());
```

Comment:
This is swallowing the underlying stacktrace, and just returning `getMessage()` (which can be null). This should probably instead be: `throw new PatchBadRequestException("error parsing the body", excIO);`